contracts/HelloWorld.abi

```diff
@@ -0,0 +1 @@
+[{"constant":false,"inputs":[{"name":"n","type":"string"}],"name":"set","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"inputs":[],"payable":false,"stateMutability":"nonpayable","type":"constructor"}]
```

Comment:
this should be deleted.

Reply:
This is used for documentation, pls refer to:
https://fisco-bcos-documentation.readthedocs.io/zh_CN/feature-python-sdk/docs/sdk/python_sdk/console.html